content/resources/help-centre/software-tips/command-line.md

```diff
@@ -0,0 +1,39 @@
+---
+title: The command line
+---
+
+## What is it?
+
+The command line, or command-line interface (CLI), is where commands or
+instructions can be passed from you, the user, to your computer. Many things
+that you do on your computer by clicking on a graphical user interface (GUI),
+can also be done in a CLI. For example, creating a new folder, or creating a
+text file. 
+
+The CLI is also known as a **CLI shell**. Sometimes simply referred to as a
+**shell**. A shell is essentially a program that we open on our computer, in
+order to type commands that we want the computer to execute. There are different
+'shells' available, depending on your operating system. You have probably seen
+or heard about `cmd.exe` or command prompt, which is a type of shell which comes
+with Windows by default. But there are other shell programs we can download and
+use, which have some added functionalities. For Windows we recommend downloading
+and using the following:
```

Comment:
This needs a little bit of work. Some of the definitions are not quite right. Read https://www.hanselman.com/blog/whats-the-difference-between-a-console-a-terminal-and-a-shell. 

We might need another way to present this document (a different title perhaps). The short of it is:

- CLI: command line interface - a text-based way of running programs (rather than graphical)
- shell: a program that a user sends input to (can be graphical, (Windows Explorer) or textual (MS-DOS))